.github/actions/compute-sha256/src/main.ts

```diff
@@ -0,0 +1,19 @@
+import * as core from "@actions/core";
+import * as fs from "fs";
+import { execSync } from "child_process";
+
+function shasum256(untrustedPath: string): string {
+  if (!fs.existsSync(untrustedPath)) {
+    throw new Error(`File ${untrustedPath} not present`);
+  }
+  const sha = execSync(`sha256sum ${untrustedPath}`).toString().split(" ")[0];
```

Comment:
Is there an API?

Reply:
Can you please let me know which one?

Reply:
There must be a built-in crypto package.

Reply:
I found this library https://github.com/brix/crypto-js which isn't updated since Sep 21. I don't have lots of confidence in shasum being implemented by some library maintainer that hasn't been updated, and the PRs arent merged.

I would rather trust what comes out of the box in Linux. 

Please let me know your concerns on using that comes Linux using exec.

Reply:
Keeping the codebase clean. When we said we wanted to port code to TS, we meant the entire code, not "emulate" the script. 
Hope that makes sense. Thanks

Reply:
Thanks, @joshuagl. I have updated to use the `crypto` from the nodejs ecosystem.